src/schema/markdown/messageParser.js

```diff
@@ -15,6 +15,14 @@ export const messageSchemaToMdMapping = {
 export const messageMdToPmMapping = {
   ...baseNodesMdToPmMapping,
   ...baseMarksMdToPmMapping,
+  image: {
+    node: 'image',
+    getAttrs: (tok) => ({
+      src: tok.attrGet('src'),
+      title: tok.attrGet('title') || null,
+      alt: (tok.children[0] && tok.children[0].content) || null,
+    }),
+  },
```

Comment:
Didn't we already support this in article editor?

Reply:
@pranavrajs Yes we support it in article editor. But in our message editor we are using this message parser that not supports images.

Reply:
Can't we combine this similar to the buildEditor method instead of making it as a 2 different config?

Reply:
@pranavrajs Done https://github.com/chatwoot/prosemirror-schema/pull/17/commits/825755b53f1ef4ae14fac2393b53a02e14ce21e0